kolena/_experimental/object_detection/dataset.py

```diff
@@ -305,14 +321,21 @@ def _compute_metrics(
         )
 
 
+def _safe_get_label(obj: object) -> Optional[str]:
+    if hasattr(obj, "label"):
+        return str(obj.label)
+
+    return None
+
+
 def _check_multiclass(ground_truth: pd.Series, inference: pd.Series) -> bool:
-    try:
-        labels = {x.label for x in itertools.chain.from_iterable(_filter_null(ground_truth))}.union(
-            {x.label for x in itertools.chain.from_iterable(_filter_null(inference))},
-        )
-        return len(labels) >= 2
-    except AttributeError:
-        return False
+    labels = {_safe_get_label(gt) for gt in itertools.chain.from_iterable(_filter_null(ground_truth))}.union(
+        {_safe_get_label(inf) for inf in itertools.chain.from_iterable(_filter_null(inference))},
+    )
+    if None in labels:
+        return len(labels) >= 3
+
+    return len(labels) >= 2
```

Comment:
This is now more robust to the case where some objects in the ground truths/inferences are unlabeled. Previously, any unlabeled objects would cause us to treat this as a single class problem.